.circleci/config.yml

```diff
@@ -0,0 +1,221 @@
+      - *start_sauce_connect
+      - *start_mes_aides
+      - *wait_for_sauce_connect
+      - *wait_for_mes_aides
+      - run:
+          name: Watai base suite
+          command: node_modules/.bin/watai test/integration/base-suite --config '{"driverCapabilities":{"browserName":"chrome","version":"latest"}}'
+      - run:
+          name: Watai family suite
+          command: node_modules/.bin/watai test/integration/family-suite  --config '{"driverCapabilities":{"browserName":"chrome","version":"latest"}}'
+  # test_selenium_edge:
+  #   <<: *defaults
+  #   steps:
+  #     - checkout
+  #     - attach_workspace:
+  #         at: ~/mes-aides-ui
+  #     - *configure_watai
+  #     - *download_sauce_connect
+  #     - *start_sauce_connect
+  #     - *start_mes_aides
+  #     - *wait_for_sauce_connect
+  #     - *wait_for_mes_aides
+  #     - run:
+  #         name: Watai base suite
+  #         command: node_modules/.bin/watai test/integration/base-suite --config '{"driverCapabilities":{"platform":"Windows 10","browserName":"MicrosoftEdge","version":"latest"}}'
+  #     - run:
+  #         name: Watai family suite
+  #         command: node_modules/.bin/watai test/integration/family-suite  --config '{"driverCapabilities":{"platform":"Windows 10","browserName":"MicrosoftEdge","version":"latest"}}'
+  # test_selenium_firefox:
+  #   <<: *defaults
+  #   steps:
+  #     - checkout
+  #     - attach_workspace:
+  #         at: ~/mes-aides-ui
+  #     - *configure_watai
+  #     - *download_sauce_connect
+  #     - *start_sauce_connect
+  #     - *start_mes_aides
+  #     - *wait_for_sauce_connect
+  #     - *wait_for_mes_aides
+  #     - run:
+  #         name: Watai base suite
+  #         command: node_modules/.bin/watai test/integration/base-suite --config '{"driverCapabilities":{"browserName":"firefox","version":"latest"}}'
+  #     - run:
+  #         name: Watai family suite
+  #         command: node_modules/.bin/watai test/integration/family-suite  --config '{"driverCapabilities":{"browserName":"firefox","version":"latest"}}'
+  deploy:
+    machine:
+      enabled: true
+    steps:
+      - run:
+          name: Deploy on production
+          command: |
+            ssh root@metal.mes-aides.gouv.fr
+
+workflows:
+  version: 2
+  build_and_test:
+    jobs:
+      - build
+      - wait_openfisca
+      - jshint:
+          requires:
+            - build
+      - test_mocha:
+          requires:
+            - jshint
+      - test_karma:
+          requires:
+            - jshint
+      - test_selenium_chrome:
+          requires:
+            - jshint
+            - wait_openfisca
+      # - test_selenium_edge:
+      #     requires:
+      #       - jshint
+      #       - wait_openfisca
+      # - test_selenium_firefox:
+      #     requires:
+      #       - jshint
+      #       - wait_openfisca
+      - deploy:
+          requires:
+            - test_mocha
+            - test_karma
+            - test_selenium_chrome
```

Comment:
On pourrait regrouper les tests dans un _job_ `test` pour y faire référence ici.

Reply:
Je préfère garder plusieurs jobs pour le parallélisme, c'est comme ça dans la [doc](https://circleci.com/docs/2.0/workflows/#fan-outfan-in-workflow-example)
J'ai un peu abstrait comme tu le suggérais avec `package.json`, ça devient (en gros) : 
```
build → lint → test (unit) → test (functional)
```

Comme ça, c'est un peu moins lié à la techno